base/interactiveutil.jl

```diff
@@ -64,7 +64,7 @@ function edit(path::AbstractString, line::Integer=0)
     nothing
 end
 
-function edit(m::Method)
+function edit(m::TypeMapEntry)
```

Comment:
I would really like to figure out how to switch this back. Needing to program with `TypeMapEntry` this much seems to go back on the promise of increasing the encapsulation of method caches. We don't want to see packages depending on the fields of this type, which they certainly will if it's returned by functions like `methods` and `methodswith`.